src/platforms/java/common/performance/index.mdx

```diff
@@ -3,10 +3,6 @@ title: Performance Monitoring
 description: "Learn more about how to configure our Performance integrations to get the best experience out of it."
 ---
 
-<Alert level="warning" title="Note">
```

Comment:
why are we removing the warning notes? they are just about the version, not really about being a prerelease

Reply:
warning notes are brought back now.